nuxeo-web-ui-ftest/webdriver/test/pages/ui/group.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+export default class Group {
```

Comment:
This should extend the `Base` class, since it is already duplicating some of its code.